cpp/order/LocalWl.cc

```diff
@@ -34,13 +34,13 @@ void LocalWl::computeYlm(const float theta, const float phi, std::vector<std::co
     for (typename fsph::PointSPHEvaluator<float>::iterator iter(sph_eval.begin_l(m_l, 0, false));
          iter != sph_eval.end(); ++iter)
     {
-        Ylm[(j + m_l) % (2 * m_l + 1)] = *iter;
+        // Manually add the Condon-Shortley phase, (-1)^m, to positive odd m
+        float phase = (j % 2 == 1) ? -1 : 1;
+        Ylm[m_l + j] = phase * (*iter);
+        if (j > 0)
+            Ylm[m_l - j] = std::conj(*iter);
```

Comment:
Is the lack of conjugation from before also an error?

Reply:
Wow OK yes it is. That's pretty egregious.

Reply:
I know... but it's fixed now.